packages/fxa-settings/src/lib/sensitive-data-client.ts

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export class SensitiveDataClient {
+  private sensitiveData: { [key: string]: string } = {};
+  private key: CryptoKey | null = null;
+  private iv: Uint8Array | undefined;
+
+  constructor() {
+    this.sensitiveData = {};
+  }
+
+  async init() {
```

Comment:
nit: Is there a reason to do this in a seperate function as opposed to the contructor?